storage/config/random_io.job

```diff
@@ -5,12 +5,13 @@ direct=1
 sync=1
 fsync_on_close=1
 time_based=1
-runtime=30
+runtime=3600
```

Comment:
Missed this in the last review.
Do you want to run the test this long? It does not show big diff between 5 mins / 2 hours, at least for 16K case.